ManicDiggerLib/Server/Mods/War/WarBlocks.cs

```diff
@@ -187,21 +187,33 @@ public void Start(ModManager m)
 			               });
 			m.SetBlockType(161, "Flag", new BlockType()
 			               {
-			               	AllTextures = "WhiteCloth",
-			               	DrawType = DrawType.Solid,
-			               	WalkableType = WalkableType.Solid,
-			               	Sounds = solidSounds,
-			               });
+							AllTextures = "FlagWhite",
+							DrawType = DrawType.Fence,
+							WalkableType = WalkableType.Empty,
+							Sounds = solidSounds,
+							});
 			m.SetBlockType(162, "BluePodest", new BlockType()
 			               {
-			               	AllTextures = "BlueCloth",
+			               	TextureIdTop = "BlueCloth",
+			               	TextureIdBack = "BluePodest",
```

Comment:
Here the shorthand property `SideTextures` could be used to set all four sides at once.